lib/CMakeLists.txt

```diff
@@ -296,6 +298,7 @@ endif()
 
 # Link in additional libraries
 target_link_libraries(libfacter PRIVATE
+    ${CPPHOCON_LIBRARIES}
```

Comment:
Will libfacter eventually be reading from config? i.e. when running as Ruby? I don't see a current use for linking here, but that seems like something we'd need.